packages/web-app-files/src/mixins/spaces/actions/disable.js

```diff
@@ -0,0 +1,68 @@
+import { mapActions } from 'vuex'
+
+export default {
+  computed: {
+    $_disable_items() {
+      return [
+        {
+          name: 'delete',
+          icon: 'forbid-2',
+          label: () => {
+            return this.$gettext('Disable')
+          },
+          handler: this.$_disable_showModal,
+          isEnabled: (space) => {
+            return !(space.root?.deleted?.state === 'trashed')
+          },
+          componentType: 'oc-button',
+          class: 'oc-files-actions-delete-trigger'
+        }
+      ]
+    }
+  },
+  methods: {
+    ...mapActions([
+      'createModal',
+      'hideModal',
+      'setModalInputErrorMessage',
+      'showMessage',
+      'toggleModalConfirmButton'
+    ]),
+
+    $_disable_showModal(space) {
```

Comment:
Same for the methods part of the action mixins, they expect a function named `trigger` (prefixed by the namespace) which is expecting an array of resources (spaces).